fala/apps/adviser/templatetags/adviser_extras.py

```diff
@@ -30,3 +30,8 @@ def query_to_dict(_context, value, prop=None):
 @pass_context
 def to_fala_page_url(context, value, url_path):
     return "%s?%s" % (url_path, urlparse(value).query)
+
+
+@library.filter
+def google_map_params(location):
```

Comment:
```suggestion
def google_map_params(location):
    return {"api": 1, "query": f"{location['address']} {location['postcode']}"}
```

Reply:
I'm not sure that there is any merit in your solution over mine?

Reply:
This feels more complicated than my version

Reply:
Yes it's tiny use here, I agree, but I'd always recommend using f-string. f-strings support the full power of Python expressions, allowing you to embed variables, function calls, and other expressions directly within the string. 
This is normal in Python to use f-string where possible over using concatenation strings. I'd recommend the use of f-string anytime over concatenation strings where possible. 

Sorry I personally disagree on the readability, that doesn't mean I'm write of course and that comes down to personal preference on some cases but consider the following. f-strings make the code more readable by allowing you to embed expressions directly within the string, surrounded by curly braces ({}). This eliminates the need for complex string concatenation and helps improves code clarity. f-strings also help reduce boilerplate code by providing a concise syntax for string formatting. 

Whenever I see `+ "" +`  in code, I cry a little bit on the inside :(

Reply:
This looks to have been resolved now?

fwiw I definitely agree that string concatenation in Python code sets off the little alarm bells for me. I would always go with a `string.format()` or, if I were less old-fashioned, the newer and better f-strings :D